### ISSUE TITLE
The Future of Creators

### ISSUE THREAD

**Issue by jeff-zucker** [solid/process#314]
To Current Creators : 

* @Vinnl 
* @VirginiaBalseiro 
* @kay-kim
* @NSeydoux 
* @KyraAssaad 
* @TallTed 
* @jeff-zucker 

The Solid Team is in the process of re-evaluating roles on the team including the Creator role.  So, I (Jeff) have been tasked with finding out where we currently stand.  Solidproject.org is in need of some TLC and, in my opinion, is overdue for a planning session/roadmap of its future. So to get us started, can each of you answer the questions below.  I'll start.

### 1. Are you interested in continuing as a Creator on the Solid Team?

Yes.

### 2. If yes, please describe the kinds of tasks you'd like to work on and an estimate of how much time you can put in

I want  to finalize the things I have in progress - the glossary page and the repository page (a page listing where the repos formerly under github.com/solid have gone) and continue things I've been doing - maintain the tools/libraries page; moderate the forum; review incoming PRs; work towards RDFa versions of the teams page, the apps page, the pods page, the stakeholders page, all of which are essentially based on data.  I'd also like to participate in planning of future directions of the site.

I can commit to 4 hours a week, possibly more as needed.

### 3. Do you know much about Jekyll and the SP.org build process?

I can figure out how to create a new page but getting it onto the menus is not obvious.  If we're going to keep using this tool, we should document the basics.

### 4. Would you be willing to participate in a planning session to figure out future directions?

Yes.

**Comment by Vinnl** [comment]
> 1. Are you interested in continuing as a Creator on the Solid Team?

Depends on the work :)

> 2. If yes, please describe the kinds of tasks you'd like to work on and an estimate of how much time you can put in

I'm OK occasionally reviewing a small PR.

> 3. Do you know much about Jekyll and the SP.org build process?

Only the bare minimum.

> 4. Would you be willing to participate in a planning session to figure out future directions?

I don't think I have the time for that, but probably wouldn't be of much value anyway given the limited amount of effort I can put into action items.

**Comment by NSeydoux** [comment]
> 1. Are you interested in continuing as a Creator on the Solid Team?

I would be happy to, but in all fairness I haven't done any active contributions in months.

> 2. If yes, please describe the kinds of tasks you'd like to work on and an estimate of how much time you can put in

I can review some PRs and/or provide feedback on technical writing for accuracy.

3. Do you know much about Jekyll and the SP.org build process?

Much is a strong word, I am vaguely aware of where some of the content is and how to navigate it.

4. Would you be willing to participate in a planning session to figure out future directions?

Probably not, or rather I doubt I'll have time to do any meaningful work based on what is discussed in that planning session, so I'm not sure my presence would bring much value.

**Comment by VirginiaBalseiro** [comment]
> Are you interested in continuing as a Creator on the Solid Team?
On principle yes, but I need more definition on what being a Creator entails. My current responsibilities span multiple things from specs editing/authoring to CoC enforcing, among other things.

> If yes, please describe the kinds of tasks you'd like to work on and an estimate of how much time you can put in
All of the above. The time I can put in is any free time I have outside work, which is limited, but I am committed to do as much as I can. 

Do you know much about Jekyll and the SP.org build process?
> No, but I have learnt worse things :) 

> Would you be willing to participate in a planning session to figure out future directions?
Yes.

**Comment by kay-kim** [comment]
1. Are you interested in continuing as a Creator on the Solid Team?

Yes. But, at least in the near future (next quarter or so), will be very
limited in time/contributions (heh heh ... kind of like this whole year).
2. If yes, please describe the kinds of tasks you'd like to work on and an
estimate of how much time you can put in

Refactoring/reorganizing/creating/rewriting content.  Unfortunately,
participation will be very limited in the next quarter or so.
3. Do you know much about Jekyll and the SP.org build process?

Not really.  It's been pretty much ad hoc figuring out as I go along -- but
I've only been contributing intermittently.
4. Would you be willing to participate in a planning session to figure out
future directions?

Because of limited time, it probably won't make sense for me to attend
these initial planning meetings.

-- 

*Kay Kim, *Technical Writer

Contact  | ***@***.***

Connect | *LinkedIn <https://www.linkedin.com/in/kaykim00/>*, *WebID
<https://kay.inrupt.net/profile/card#me>*

Explore  |
*www.inrupt.com <http://www.inrupt.com/>*

-- 
This e-mail, and any attachments thereto, is intended only for use by the 
addressee(s) named herein and may contain legally privileged, confidential 
and/or proprietary information. If you are not the intended recipient of 
this e-mail (or the person responsible for delivering this document to the 
intended recipient), please do not disseminate, distribute, print or copy 
this e-mail, or any attachment thereto. If you have received this e-mail in 
error, please respond to the individual sending the message, and 
permanently delete the email.

**Comment by TallTed** [comment]
> 1. Are you interested in continuing as a Creator on the Solid Team?

Yes. (It seems worth noting that I did not self-nominate for this role; I was named as such based on the activities I was, and continue to, participating in.)

> 2. If yes, please describe the kinds of tasks you'd like to work on and an estimate of how much time you can put in

I'm best suited to the things I've been doing, i.e., reviewing grammar, punctuation, MD, HTML, and the like. My primary programming language is English, supplemented by MD, HTML, and similar markup languages.

Available time varies, from a few minutes to several hours any given week, with little predictability due to the variance in activity dedicated to my employer.

> 3. Do you know much about Jekyll and the SP.org build process?

Nothing (yet).

> 4. Would you be willing to participate in a planning session to figure out future directions?

Sure. Optimally, this would fit into an existing Solid-focused time slot, as most such do not conflict with my other commitments. A doodle or similar poll to find a mutually convenient time slot would be a good fallback, if not first effort.

**Comment by KyraAssaad** [comment]
Sorry I missed this before, here are my responses. 

> 1. Are you interested in continuing as a Creator on the Solid Team?
Yes, but not indefinitely. Like @TallTed, I also was named a Creator based on the activities I was participating in at the time.

> 2. If yes, please describe the kinds of tasks you'd like to work on and an estimate of how much time you can put in
I would like to be involved in the redesign of the SP.org site. I can bring my UX, UI and web design skills to offer. I worked with @MarrelleBailey in the past to do usability research on the existing site; there is a cache of interviews done that have valuable information to inform a redesign. I would be able to contribute my visual and interaction skills as well as basic HTML/CSS skills, but would not be able to contribute on anything more technical than that.

> 3. Do you know much about Jekyll and the SP.org build process?
I understand the process of how it works but none of the implementation or technical details.

> 4. Would you be willing to participate in a planning session to figure out future directions?
Yes

**Comment by michielbdejong** [comment]
> 1. Are you interested in continuing as a Creator on the Solid Team?

I'm interested to start! :) (see also #317)

> 2. If yes, please describe the kinds of tasks you'd like to work on and an estimate of how much time you can put in

I would like to make sure that different types of information are findable for people who play different roles, with different needs that bring them to the website, and different knowledge/skill background that influence how they will understand the text we put on there). And improving the quality of the website in that respect.

Also making sure the information stays up-to-date, accurate and complete.

I can commit to 4 hours a month, possibly more as needed.

> 3. Do you know much about Jekyll and the SP.org build process?

No but if we run into any issues with that, I'm sure we'll figure it out :)

> 4. Would you be willing to participate in a planning session to figure out future directions?

Yes.
